vm/devices/vmbus/vmbus_server/src/lib.rs

```diff
@@ -231,6 +231,11 @@ impl EventPort for ChannelEvent {
 pub struct SavedState {
     #[mesh(1)]
     server: channels::SavedState,
+    // Indicates if the lost synic bug is fixed or not. By default it's false.
+    // During the restore process, we check if the field is not true then
+    // unstick_channels() function will be called to mitigate the issue.
+    #[mesh(2)]
+    lost_synic_bug_fixed: bool,
```

Comment:
Just double checking the behavior: what happens if:

* Service from OpenHCL w/ `lost_synic_bug_fixed` -> OpenHCL w/o `lost_synic_bug_fixed`
* Service from OpenHCL w/o `lost_synic_bug_fixed` -> OpenHCL w/ `lost_synic_bug_fixed`

For the first case, the restore side won't know how to handle the `lost_synic_bug_fixed` field. Does the restore side just ignore the extra data?
For the second case, does the restore side default the field to `false` without complaint? (I think so, based on your comment)

Reply:
I was educated offline:

new -> old: unknown field is ignored
old -> new: default value kicks in

No conerns.